content/en/docs/concepts/services-networking/endpointslice.md

```diff
@@ -0,0 +1,97 @@
+---
+reviewers:
+- freehan
+title: Endpoint Slice
+feature:
+  title: Endpoint Slices
+  description: >
+    Scalable tracking of network endpoints in a Kubernetes cluster.
+
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< glossary_definition term_id="endpointslice" length="short" >}}
+
+Endpoint Slices provide a simple way to track network endpoints within a
+Kubernetes cluster. Currently in alpha, they are intended to eventually replace
+the current Endpoints resource within Kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivation
+
+The current Endpoints API has provided a simple and straightforward way of
+tracking network endpoints in Kubernetes. Unfortunately as Kubernetes clusters
+and Services have gotten larger, limitations of that API became more visible.
+Most notably, those included challenges with scaling to larger numbers of
+network endpoints. 
+
+Since all network endpoints for a Service were stored in a single Endpoints
+resource, those resources could get quite large. That affected the performance
+of Kubernetes components like kube-proxy and resulted in significant amounts of
```

Comment:
kube-proxy isn't actually the main victim of it - it's more like master-control-plane (etcd, apiserver)

Reply:
Updated, thanks!